lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -9,24 +9,27 @@ use Path::Class qw();
 class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
     is => 'Command::V2',
     has_input => [
+        model => {
```

Comment:
Perhaps.  So far the single test user using this tool has been working with models.  Internally, tool really works with a specified bam file, wherever it comes from.  As a convenience, I thought it would be helpful if the user could just specify a model directly on the command line.  But, if the user wanted to "bypass" the system, or work with a specialized data set, they could alternatively specify a "bam-file" directly.

I'm not sure which "data currency" works best for the end-users: models and/or builds.  Do you guys think inputting a build rather than a model is the best way to proceed?